Source/WebKit/UIProcess/WebProcessPool.h

```diff
@@ -883,6 +883,9 @@ class WebProcessPool final
     RefPtr<StorageAccessUserAgentStringQuirkObserver> m_storageAccessUserAgentStringQuirksDataUpdateObserver;
     RefPtr<StorageAccessPromptQuirkObserver> m_storageAccessPromptQuirksDataUpdateObserver;
 #endif
+#if PLATFORM(IOS_FAMILY) && !PLATFORM(MACCATALYST)
+    float m_displayBrightness { 0 };
```

Comment:
Unclear how this relates to this patch exactly.

Reply:
It is a very good point to break this up in several patches. I will address the concerns in follow-up patches.

Thanks for reviewing, Chris!